CONTRIBUTING.md

```diff
@@ -205,6 +205,11 @@ Or if you're running Fedora 34 or later, it can be installed directly with
 dnf install mqttcli
 ```
 
+## `MQTT_DEBUG`
```

Comment:
Maybe change the section title to a description (that can render nicely in table
of contents, for example)?

```suggestion
## Debugging MQTT traffic
```